torch_xla/_dynamo/dynamo_bridge.py

```diff
@@ -805,7 +805,8 @@ def is_node_supported(self, submodules, node: torch.fx.Node) -> bool:
 
 def extract_compiled_graph_helper(xla_model: torch.fx.GraphModule, xla_args):
   if _args_on_cuda(xla_args):
-    xla_args = tuple(_maybe_move_tensors_to_device(xla_args, xm.xla_device()))
+    xla_args = tuple(
+        _maybe_move_tensors_to_device(xla_args, torch_xla.device()))
```

Comment:
Make this one line?

Reply:
This goes over 80 character limit by our linter.